jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -70,6 +68,20 @@ public synchronized static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
 
+	/**
+	 * Queries if CRIU Security provider needs to be disabled
+	 * when CRIU support is enabled.
+	 *
+	 * @return true if disabled, otherwise false
+	 */
+	public static boolean disableCRIUSecProvider() {
```

Comment:
As I thought about this some more, I came to the conclusion that uses of this new API would be easier to follow without the negation; that is this method should be named `enableCRIUSecProvider()` (or similar) and answer whether `CRIUSECProvider` **_should_** be used.

Also, please add and check for this option as well (for consistency and to provide the ability to override `-XX:-CRIUSecProvider` specified by an environment variable or earlier command-line option).
```c
#define VMOPT_XXENABLECRIUSECPROVIDER "-XX:+CRIUSecProvider"
```
It might even help to replace `J9VM_CRIU_DISABLE_CRIU_SEC_PROVIDER` with the positive flag `J9VM_CRIU_ENABLE_CRIU_SEC_PROVIDER`.

Reply:
Updated